src/base/bittorrent/session.cpp

```diff
@@ -4493,22 +4493,22 @@ void Session::handlePeerBlockedAlert(const lt::peer_blocked_alert *p)
     QString reason;
     switch (p->reason) {
     case lt::peer_blocked_alert::ip_filter:
-        reason = tr("due to IP filter.", "this peer was blocked due to ip filter.");
+        reason = tr("IP filter", "this peer was blocked. Reason: IP filter.");
         break;
     case lt::peer_blocked_alert::port_filter:
-        reason = tr("due to port filter.", "this peer was blocked due to port filter.");
+        reason = tr("port filter", "this peer was blocked. Reason: port filter.");
         break;
     case lt::peer_blocked_alert::i2p_mixed:
-        reason = tr("due to i2p mixed mode restrictions.", "this peer was blocked due to i2p mixed mode restrictions.");
+        reason = tr("i2p mixed mode restrictions", "this peer was blocked. Reason: i2p mixed mode restrictions.");
```

Comment:
Shouldn't "i2p" be written as "I2P"?

Reply:
@glassez

I guess so, that is how it is stylized in various important sources:
https://en.wikipedia.org/wiki/I2P
https://geti2p.net/

Fixed, and also brought the form more in line with that of `TCP` and `uTP.` (the `don't translate` comment).